samples/plugins/mysql_odbc/connectionResolver.tdr

```diff
@@ -13,6 +13,7 @@
           <attr>username</attr>
           <attr>password</attr>
           <attr>port</attr>
+          <attr>sslmode</attr>
```

Comment:
Have you tested connection with SSL enabled?  I think sslcert should be required too.

Reply:
I have, it works and I confirmed with wireshark that the traffic is encrypted. No cert requried.